src/components/CopyTextToClipboard.js

```diff
@@ -39,40 +28,24 @@ class CopyTextToClipboard extends React.Component {
     }
 
     copyToClipboard() {
-        if (this.props.isDelayButtonStateComplete) {
-            return;
-        }
         Clipboard.setString(this.props.text);
-        this.props.toggleDelayButtonState(true);
     }
 
     render() {
         return (
-            <Text
+            <PressableWithDelayToggle
+                text={this.props.text}
+                tooltipText={this.props.translate('reportActionContextMenu.copyToClipboard')}
+                tooltipTextChecked={this.props.translate('reportActionContextMenu.copied')}
+                icon={Expensicons.Copy}
+                textStyles={this.props.textStyles}
                 onPress={this.copyToClipboard}
-                style={[styles.flexRow, styles.cursorPointer]}
-                suppressHighlighting
```

Comment:
Coming from https://github.com/Expensify/App/issues/24591

We are missing the prop `suppressHighlighting` if the `PressableWithDelayToggle` uses the `Text` component.